addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/ServerSideIncludeScanRule.java

```diff
@@ -203,4 +206,9 @@ public int getCweId() {
     public int getWascId() {
         return 31;
     }
+
+    @Override
+    public List<Alert> getExampleAlerts() {
+        return List.of(buildAlert("profile", SSI_UNIX, patternSSIUnix.toString()).build());
```

Comment:
The evidence is not the pattern but what was matched by the pattern.

Reply:
Oops the String Builder toString call mislead me 🤦‍♂️